### DISCUSSION TITLE
Latest commit that uses xformer 0.014dev for auto1111?

### DISCUSSION THREAD

**Discussion by giteeeeee** [TheLastBen/fast-stable-diffusion#1534] [Q&A]
Just want to test a few things out.

**Comment by TheLastBen** [comment]
I didn't keep track of that

**Reply by giteeeeee** [reply]
Thanks, guess I'll just dig a little. Though was it roughly about 1 month ago?

**Comment by giteeeeee** [comment]
@TheLastBen Hey I was able to find the 0.0.14 xformer in the commit 500611b. However, I'd like to implement this xformer into your latest repo for auto1111 webui.
I tried replacing "precompiled/T4/T4" in your latest commit by that in the older commit, but the webui seems to still use 0.0.16 after reinstalling.
I also downloaded the .14 xformer whl from the older commit, but unsure of where to place it.
Is it possible that you can teach me how to replace the files to make the xformer work for webui?

**Reply by TheLastBen** [reply]
xformers is included in the dependencies file extracted in the 2nd cell, you need to install xformers after running that cell
